lib/ClangImporter/ImportType.cpp

```diff
@@ -2356,12 +2356,17 @@ ClangImporter::Implementation::importParameterType(
                  dyn_cast<clang::TemplateTypeParmType>(paramTy)) {
     swiftParamTy = findGenericTypeInGenericDecls(
         *this, templateParamType, genericParams, attrs, addImportDiagnosticFn);
-  } else if (auto refType = dyn_cast<clang::ReferenceType>(paramTy)) {
+  } else if (auto refType = dyn_cast<clang::ReferenceType>(paramTy)) {    
```

Comment:
Looks like we introduced some whitespace at the end here.

Reply:
that's resolved in updated PR